src/MSALWrapper/AuthFlow/Broker.cs

```diff
@@ -139,6 +158,19 @@ public async Task<AuthFlowResult> GetTokenAsync()
             return new AuthFlowResult(null, this.errors, this.GetType().Name);
         }
 
+        [DllImport("kernel32.dll")]
+        private static extern IntPtr GetConsoleWindow();
+
+        [DllImport("user32.dll", ExactSpelling = true)]
+        private static extern IntPtr GetAncestor(IntPtr windowsHandle, GetAncestorFlags flags);

```

Comment:
In the corresponding MSAL issue there were some additional comments above this function. Should we replicate those for completeness? 🤔

Reply:
added the comments.